src/natcap/invest/delineateit/delineateit_core.pyx

```diff
@@ -0,0 +1,85 @@
+# cython: language_level=2
```

Comment:
Okay! As I understand it, we only need this because `language_level=2` means it will be compiled using python2 semantics, and python2 doesn't default to UTF-8. Why not use `language_level=3`?

Reply:
Ok! Changed to `language_level=3` and removed the `coding: utf-8` since it shouldn't be needed